addons/website_mail_channel/static/src/snippets/s_channel/000.js

```diff
@@ -27,7 +27,6 @@ publicWidget.registry.Channel = publicWidget.Widget.extend({
                 self.$target.find(".js_mg_follow_form").remove();
             }
             self._toggleSubscription(data.is_member ? 'on' : 'off', data.email);
-            self.$target.removeClass('d-none');
```

Comment:
I don't think you want to remove the fact that the snippet is hidden while it is not loaded yet: the d-none class should just not be added by the option but in the destroy of this public widget, then I think it would work?